docs/guides/upgrading-to-1-0.md

```diff
@@ -92,6 +95,27 @@ var oldBullettedList = {
 
 ```
 
+### Formats
+
+Previous versions of Quill allowed customization of formats in dropdown selectors in the toolbar by using `value` attributes. That is, `<select class="ql-size"><option value="10px">10</option>[...]</select>` worked to provide custom size options. This is no longer supported. To restore this functionality, you can use the following:
```

Comment:
What changed was Quill defaulted to using classes instead of styles, and those classes were named descriptively instead of `ql-size-10px`. The dropdowns still work the same.